spec/requests/packages_spec.rb

```diff
@@ -553,6 +553,25 @@
     end
   end
 
+  describe 'getting resources related to package times out' do
+    before do
+      expect_any_instance_of(Faraday::Connection).to receive(:get).and_return(Map(body: { 'errors': [{ 'title': 'Request timed out', 'detail': 'Timed out getting https://api.ebsco.io/rm/rmaccounts/apidvcorp/vendors/19/packages/796/titles?count=25&offset=1&orderby=relevance&resourcetype=all&search=aa&searchfield=titlename&selection=all' }], 'jsonapi': { 'version': '1.0' } }.to_json, status: 408))
+    end
+
```

Comment:
Nice to know how to mock this!

Reply:
Yes, it was pretty challenging. Just updated the `learning` part with all learning that I had to go through :)